### ISSUE TITLE
Running psiturk without aws

### ISSUE THREAD

**Issue by Tal-Golan** [NYUCCL/psiTurk#355]
I want to use PsiTurk with prolific.ac instead of mturk. Is there a way to circumvent the aws login (it results with 'The security token included in the request is invalid.') or does one must have aws credentials even if mturk won't be used?

**Comment by deargle** [comment]
What command are you running that returns that error message?

**Comment by Tal-Golan** [comment]
Just running psiturk from the shell, within an experiment's folder. In the past, I managed to get through with a warning, but since I updated psiturk this doesn't work anymore.

```
$ psiturk
An error occurred (UnrecognizedClientException) when calling the GetAccountBalance operation: The security token included in the request is invalid.
```

**Comment by deargle** [comment]
if you just want to launch the server, you can do `psiturk server <command>` all from bash -- is there anything you need the _shell_ for in particular? I can't think of what it would be useful for if you're not using aws, but then again I have never used it without aws creds.

``` bash
(psiturk-default-3.6) deargle@deargle-VirtualBox ~/projects/psiturk-default $ psiturk server on
Experiment server launching...
Now serving on http://0.0.0.0:22362
(psiturk-default-3.6) deargle@deargle-VirtualBox ~/projects/psiturk-default $ ps aux | grep psiturk
deargle  12322  0.0  0.0   4504   784 pts/3    S    10:20   0:00 /bin/sh -c /home/deargle/.virtualenvs/psiturk-default-3.6/bin/python3.6 '/home/deargle/projects/psiTurk/psiturk/experiment_server.py'
deargle  12323  2.4  0.5  93300 23076 pts/3    S    10:20   0:00 gunicorn: master [psiturk_experiment_server_3b8ab255cb7e]
deargle  12326  8.5  1.4 218224 58308 pts/3    S    10:20   0:01 gunicorn: worker [psiturk_experiment_server_3b8ab255cb7e]
deargle  12338  0.0  0.0  14224   968 pts/3    S+   10:20   0:00 grep --color=auto psiturk
(psiturk-default-3.6) deargle@deargle-VirtualBox ~/projects/psiturk-default $ psiturk server off
Shutting down experiment server at pid b'12323'...
Please wait. This could take a few seconds.
Killed
(psiturk-default-3.6) deargle@deargle-VirtualBox ~/projects/psiturk-default $

```

**Comment by deargle** [comment]
I just released 2.3.2 which gives a more helpful tips-and-tricks error message for non-aws users instead of crashing and burning. Let me know if there's anything I'm missing.

**Comment by Tal-Golan** [comment]
Thanks, using  `psiturk server <command> `is a good solution.
